<ui version="4.0" stdsetdef="1" >
  <class>FindDialog</class>
  <comment>*********************************************************************
  **
  ** Copyright (C) 1992-$THISYEAR$ Trolltech AS. All rights reserved.
  **
  ** This file is part of the $MODULE$ of the Qt Toolkit.
  **
  ** $LICENSE$
  **
  ** This file is provided AS IS with NO WARRANTY OF ANY KIND, INCLUDING THE
  ** WARRANTY OF DESIGN, MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
  **
  *********************************************************************</comment>
  <widget class="QDialog" name="FindDialog" >
    <property name="objectName" >
      <string notr="true" >FindDialog</string>
    </property>
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>414</width>
        <height>139</height>
      </rect>
    </property>
    <property name="sizePolicy" >
      <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>4</vsizetype>
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle" >
      <string>Qt Linguist</string>
    </property>
    <property name="whatsThis" stdset="0" >
      <string>This window allows you to search for some text in the translation source file.</string>
    </property>
    <layout class="QHBoxLayout" >
      <property name="objectName" >
        <string notr="true" >unnamed</string>
      </property>
      <item>
        <layout class="QVBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <item>
            <layout class="QHBoxLayout" >
              <property name="objectName" >
                <string notr="true" >unnamed</string>
              </property>
              <item>
                <widget class="QLabel" name="findWhat" >
                  <property name="objectName" >
                    <string notr="true" >findWhat</string>
                  </property>
                  <property name="text" >
                    <string>Find what:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLineEdit" name="led" >
                  <property name="objectName" >
                    <string notr="true" >led</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>Type in the text to search for.</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="spacer3" >
              <property name="sizeHint" >
                <size>
                  <width>0</width>
                  <height>0</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QGridLayout" >
              <property name="objectName" >
                <string notr="true" >unnamed</string>
              </property>
              <item row="0" column="0" >
                <widget class="QCheckBox" name="sourceText" >
                  <property name="objectName" >
                    <string notr="true" >sourceText</string>
                  </property>
                  <property name="text" >
                    <string>Source texts</string>
                  </property>
                  <property name="checked" >
                    <bool>true</bool>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>Source texts are searched when checked.</string>
                  </property>
                </widget>
              </item>
              <item row="0" column="1" >
                <widget class="QCheckBox" name="matchCase" >
                  <property name="objectName" >
                    <string notr="true" >matchCase</string>
                  </property>
                  <property name="text" >
                    <string>Match case</string>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>Texts such as 'TeX' and 'tex' are considered as different when checked.</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="0" >
                <widget class="QCheckBox" name="translations" >
                  <property name="objectName" >
                    <string notr="true" >translations</string>
                  </property>
                  <property name="text" >
                    <string>Translations</string>
                  </property>
                  <property name="checked" >
                    <bool>true</bool>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>Translations are searched when checked.</string>
                  </property>
                </widget>
              </item>
              <item row="2" column="0" >
                <widget class="QCheckBox" name="comments" >
                  <property name="objectName" >
                    <string notr="true" >comments</string>
                  </property>
                  <property name="text" >
                    <string>Comments</string>
                  </property>
                  <property name="checked" >
                    <bool>true</bool>
                  </property>
                  <property name="whatsThis" stdset="0" >
                    <string>Comments and contexts are searched when checked.</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
      <item>
        <layout class="QVBoxLayout" >
          <property name="objectName" >
            <string notr="true" >unnamed</string>
          </property>
          <item>
            <widget class="QPushButton" name="findNxt" >
              <property name="objectName" >
                <string notr="true" >findNxt</string>
              </property>
              <property name="text" >
                <string>Find Next</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to find the next occurrence of the text you typed in.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="cancel" >
              <property name="objectName" >
                <string notr="true" >cancel</string>
              </property>
              <property name="text" >
                <string>Cancel</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Click here to close this window.</string>
              </property>
            </widget>
          </item>
          <item>
            <spacer name="spacer2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>51</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <tabstops>
    <tabstop>led</tabstop>
    <tabstop>sourceText</tabstop>
    <tabstop>translations</tabstop>
    <tabstop>comments</tabstop>
    <tabstop>matchCase</tabstop>
    <tabstop>findNxt</tabstop>
    <tabstop>cancel</tabstop>
  </tabstops>
</ui>
